### ISSUE TITLE
Alt tags

### ISSUE THREAD

**Issue by Default4152** [CMattler/tea-cozy#1]
Great! I like that you used alt tags on your images as this improves accessibility and also gives our users a description of the image in the event they cannot access it.

Read more here: https://www.w3.org/wiki/HTML/Usage/TextAlternatives

Here is a more in-depth explanation on alt & accessibility if you are curious: https://webaim.org/techniques/alttext/
